docs/en/latest/router-radixtree.md

```diff
@@ -71,11 +71,13 @@ Here are the rules:
 |/blog/foo/gloo | `/blog/foo/*` |
 |/blog/bar | not match |
 
-#### 4. Different routes have the same uri
+#### 4. Different routes have the same `uri`
 
-When different routes have the same uri, you can set the priority attribute of the route to determine which route to match first, or add matching rules to distinguish different routes.
+When different routes have the same uri, you can set the priority field of the route to determine which route to match first, or add other matching rules to distinguish different routes.
 
-1. Different routes have the same uri and set the `priority` attribute
+Note: In matching rules, the `priority` field takes precedence over other rules. Therefore, we can set other matching rules only when the value of the `priority` field is the same.
```

Comment:
> Therefore, we can set other matching rules only when the value of the `priority` field is the same.

Can't understand it. We can set `hosts` regardless of the value of `priority`.

Reply:
Yes, this sentence seems superfluous. 
Updated.